capstone/src/main/java/com/google/sps/servlets/StudySetServlet.java

```diff
@@ -58,6 +60,11 @@
           search_sql_statement
         )
       ) {
+        String new_search_word = "%" + search_word + "%";
+        for (int i = 1; i <= 4; i++) {
```

Comment:
Generally magical number like this can be confusing to readers.

This should equal to the number of ?'s occurrences in search_sql_statement right? How about we compute it dynamically here. It can be pretty simple with Java8: https://www.baeldung.com/java-count-chars#5-using-java-8-features

Reply:
Resolved